testutil/ciao-down/ciao_down.go

```diff
@@ -77,24 +77,25 @@ func checkDirectory(dir string) error {
 	return nil
 }
 
-func prepareFlags() (memGB int, CPUs int, debug bool, uiPath string, err error) {
+func prepareFlags() (memGB int, CPUs int, debug bool, uiPath, runCmd string, err error) {
 	fs := flag.NewFlagSet("prepare", flag.ExitOnError)
 	vmFlags(fs, &memGB, &CPUs)
 	fs.BoolVar(&debug, "debug", false, "Enables debug mode")
 	fs.StringVar(&uiPath, "ui-path", "", "Host path of cloned ciao-webui repo")
+	fs.StringVar(&runCmd, "runcmd", "", "Path to a file containing additional commands to execute when preparing the VM")
```

Comment:
Yes.  I'll do this.  I also need to document and improve the language used to create these cloud-init files.  RIght now it's a bit odd.  The other thing we need is control over the ports that are opened up.  My guess is you don't want ports 3000 and 5000 opened up on CC.  I'll enter some issues for these.